StRoot/StFwdTrackMaker/StFwdTrackMaker.cxx

```diff
@@ -857,101 +1011,407 @@ int StFwdTrackMaker::Make() {
         mHistograms[ "nMcTracksFwdNoThreshold" ]->Fill( nForwardTracksNoThreshold );
     }
 
+
+    return nForwardTracks;
+} // loadMcTracks
+
+void StFwdTrackMaker::loadFcs( ) {
+    StEvent *stEvent = static_cast<StEvent *>(GetInputDS("StEvent"));
+    StFcsDb* fcsDb=static_cast<StFcsDb*>(GetDataSet("fcsDb"));
+    if ( !stEvent || !fcsDb ){
+        return;
+    }
+    StFcsCollection* fcsCol = stEvent->fcsCollection();
```

Comment:
There is no check to make sure the pointer (`fcsCol`) points to the valid object. Is this collection always expected to be present?

Reply:
check added